PluginSetup.php

```diff
@@ -37,5 +39,21 @@ public static function maybe_deactivate() {
 			wp_die();
 			exit;
 		}
+
+		if ( ! is_plugin_active( self::WOOCOMMERCE_PATH ) ) {
```

Comment:
I don't follow you here, I understand you are talking about some kind of refactoring? Which part exactly? the `wp_kses` part? just create another function and pass all the parameters? Thank you @mitogh